tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderSteps.cs

```diff
@@ -47,10 +48,10 @@ public async Task GivenOrdersExit(Table table)
             }
         }
 
-        [When(@"a GET request is made for orders")]
-        public async Task WhenAGETRequestIsMadeForOrders()
+        [When(@"a GET request is made for a list of orders with organisationId (.*)")]
+        public async Task WhenAGETRequestIsMadeForOrders(Guid organisationId)
         {
-            await _request.GetAsync(_orderUrl);
+            await _request.GetAsync($"{_orderUrl}/{organisationId}/orders");
```

Comment:
Perhaps a matter of opinion, but would you consider using the GetAsync method in a way where it utilises Flurl's features? 
What I mean is - from:
```
await _request.GetAsync($"{_orderUrl}/{organisationId}/orders")
```
To
```
await _request.GetAsync(_orderUrl, organisationId, "orders");
```

All up to you, as I said, its a matter of opinion 😄

Reply:
I've updated it with Sam's comments 😄